src/algebra/group/conj.lean

```diff
@@ -44,6 +45,18 @@ begin
   repeat {rw mul_assoc},
 end
 
+/-- `b * a * b⁻¹` as a bundled equiv. -/
+@[simps apply]
+def conj_equiv (b : α) : α ≃* α :=
+{ to_fun := λ a, b * a * b⁻¹,
+  inv_fun := λ a, b⁻¹ * a * b,
+  left_inv := λ a, by simp [←mul_assoc],
+  right_inv := λ a, by simp [←mul_assoc],
+  map_mul' := λ a b, conj_mul.symm }
+
+lemma conj_injective {x : α} : function.injective (λ (g : α), x * g * x⁻¹) :=
+(conj_equiv x).injective
```

Comment:
Is there a more general case of this lemma for general group actions in mathlib? It definitely should be there.

Reply:
I assume you mean [`mul_action.bijective`](https://leanprover-community.github.io/mathlib_docs/group_theory/group_action/group.html#mul_action.bijective)?

Reply:
Didn't have time to look for it, but you found it faster.

Reply:
I suppose the next question is "do we have a mul_action instance for group conjugation", for which the answer is "no, it would conflict with `mul_action.regular`"; followed by "do we have a mul_action _definition_ for group conjugation", for which I think the answer is yes.